app/components/CartMenuItem.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import RaisedButton from 'material-ui/RaisedButton'
+import MenuItem from 'material-ui/MenuItem'
+
+export default class CartMenuItem extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      numItems: 1,
+    }
+  }
+
+  handleQuantChange = (evt) => {
+    evt.target.value < 0 ? this.setState({numItems: 0}) : this.setState({numItems: evt.target.value})
+  };
+
+  render() {
+    const style = {margin: 12}
+    return (
+        <MenuItem >
```

Comment:
Does the px-based padding here scale responsively? Might consider using percentages or ems?